modules/axfs/src/root.rs

```diff
@@ -170,6 +170,10 @@ pub(crate) fn init_rootfs(disk: crate::dev::Disk) {
             static EXT4_FS: LazyInit<Arc<fs::ext4fs::Ext4FileSystem>> = LazyInit::new();
             EXT4_FS.init_by(Arc::new(fs::ext4fs::Ext4FileSystem::new(disk)));
             let main_fs = EXT4_FS.clone();
+        } else if #[cfg(feature = "ext4_rs")] {
```

Comment:
It will be merged by the external [StarryOS](https://github.com/Starry-OS/Starry/) in the future.